src/components/FieldLabel/FieldLabel.tsx

```diff
@@ -41,8 +41,7 @@ export const FieldLabel = React.forwardRef<HTMLLabelElement, FieldLabelProps>(
   ({ children, className, htmlFor, size = 'lg', disabled, ...other }, ref) => {
     const componentClassName = clsx(
       styles['label'],
-      size === 'md' && styles['label--md'],
-      size === 'lg' && styles['label--lg'],
+      size && styles[`label--${size}`],
```

Comment:
collapsed this into one entry since `size` is typed and can't be arbitrary. Less is more :)